### PR TITLE
Pin httpx to latest version supporting proxies

### PR DESCRIPTION
**Pull request by mmatteoo**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
+httpx<0.28
 requests>=2.18
 openai==1.27.0
```
Comment (platisd) [inline requirements.txt:3]: I believe the fix is to use a later `openai` version instead.

Comment (mmatteoo) [reply requirements.txt:3]: Yes, sorry for opening a PR here rather than on my fork, my bad 😅

**Comment by platisd** [comment]
@MatteoEgmont do you think the fact that #11 run successfully  means that the fix worked?

**Comment by mmatteoo** [comment]
@platisd yes, I tested in a repo where we use your action, running with
```
    steps:
      - uses: platisd/openai-pr-description@175d40a9cfe9c0e9809139ce28f7dc16147a1717
````

and the execution was successful

**Comment by platisd** [comment]
Nice, will merge then. Thanks for the help.

**Pull request closed without merging**